package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperid",
-  "version": "1.4.1",
+  "version": "1.4.2",
```

Comment:
Can you please revert this change? Bumping the version number is part of the release process.

Reply:
Sure, done